testsuite/features/secondary/buildhost_docker_build_image.feature

```diff
@@ -22,14 +22,17 @@ Feature: Build container images
     And I select "galaxy-registry" from "imageStore"
     And I enter "Docker" relative to profiles as "path"
     And I click on "create-btn"
-
-  Scenario: Create a simple real image profile without activation key
-    When I follow the left menu "Images > Profiles"
-    And I follow "Create"
-    And I enter "suse_real_simple" as "label"
-    And I select "galaxy-registry" from "imageStore"
-    And I enter "Docker/serverhost" relative to profiles as "path"
-    And I click on "create-btn"
+    Then I wait until I see "suse_simple" text
+     
+    #TODO: currently removed because build Docker/serverhost is failing
```

Comment:
`suse_real_simple` was never created because `docker build` command for `Docker/serverhost` fails. In this case any error was raised because there are no check